CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ Demos and examples 🤓 can be found on [fabricjs.com](http://fabricjs.com/demos
 ## 🏷️ Issue Tracker
 
 - **Before You Begin** 🎬
-  - 📌 Make sure you didn't fall into a known [**GOTCHA**](http://fabricjs.com/fabric-gotchas)
+  - 📌 Make sure you didn't fall into a known [**GOTCHA**](./GOTCHAS.md)
```

Comment:
still getting a 404 at gotchas.md

Reply:
please just leave the website link.
The advertisement banner still give use 75$ a month. If one day we need to buy
- a tablet for touch interactions
- a graphic tablet with a pen
- one of you is jobless and the laptop melt down

we have a bag of money that is project related.
(is small now, because 2 years ago i spent them for a stroller 🤷‍♂️  i was alone)

Reply:
sure 
stroller haha